libr/bin/p/bin_dex.c

```diff
@@ -156,13 +156,18 @@ static char *createAccessFlagStr(ut32 flags, AccessFor forWhat) {
 	if (!str) {
 		return NULL;
 	}
+	if (count == 0) {
+		*cp = '\0';
+		return cp;
+	}
 	for (i = 0; i < NUM_FLAGS; i++) {
 		if (flags & 0x01) {
 			const char *accessStr = kAccessStrings[forWhat][i];
 			int len = strlen (accessStr);
 			if (cp != str) {
 				*cp++ = ' ';
 			}
+			if (((cp - str) + len) >= ((count + 1) * (kLongest + 1))) return NULL;
```

Comment:
this count + 1*kLongest +1 is a bit scary, can you just put that constant outside the loop with a meaningful name instead of computing it on every iteration?